static/app/components/replays/replayCurrentUrl.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import TextCopyInput from 'sentry/components/forms/textCopyInput';
+import {useReplayContext} from 'sentry/components/replays/replayContext';
+import space from 'sentry/styles/space';
+import getCurrentUrl from 'sentry/utils/replays/getCurrentUrl';
+
+function ReplayCurrentUrl() {
+  const {currentTime, replay} = useReplayContext();
+  if (!replay) {
+    return null;
+  }
+
+  return (
+    <UrlCopyInput>
+      <TextCopyInput>{getCurrentUrl(replay, currentTime)}</TextCopyInput>
+    </UrlCopyInput>
+  );
+}
+
+const UrlCopyInput = styled('div')`
+  & input {
```

Comment:
Instead of hardcoding the selector here, we should instead export the styled components from TextCopyInput and use those as selectors. That way there's a reference to the input and buttons we are trying to style.

Reply:
will do!